idx/memory/tag_query.go

```diff
@@ -626,7 +626,7 @@ func (q *TagQuery) testByTagMatch(def *idx.Archive) bool {
 		equal := strings.Index(tag, "=")
 		if equal < 0 {
 			corruptIndex.Inc()
-			log.Error(3, "memory-idx: ID %q has tag %q in index without '=' sign", def.Id, tag)
+			log.Errorf("ID %q has tag %q in index without '=' sign", def.Id, tag)
```

Comment:
why are we no longer logging the component?

Reply:
we should be until we switch over to individual instances of logrus per module. It just looks like I forgot to place them back into that section, I'll fix it.